Formula/argyll-cms.rb

```diff
@@ -42,18 +46,29 @@ class ArgyllCms < Formula
   end
 
   def install
-    # These two inreplaces make sure /opt/homebrew can be found by the
-    # Jamfile, which otherwise fails to locate system libraries
-    inreplace "Jamtop", "/usr/include/x86_64-linux-gnu$(subd)", "#{HOMEBREW_PREFIX}/include$(subd)"
-    inreplace "Jamtop", "/usr/lib/x86_64-linux-gnu", "#{HOMEBREW_PREFIX}/lib"
-    # These two inreplaces make sure the X11 headers can be found on Linux.
-    unless OS.mac?
-      inreplace "Jamtop", "/usr/X11R6/include", HOMEBREW_PREFIX/"include"
-      inreplace "Jamtop", "/usr/X11R6/lib", HOMEBREW_PREFIX/"lib"
+    # Remove bundled libraries to prevent fallback
+    %w[jpeg png tiff zlib].each { |l| (buildpath/l).rmtree }
+
+    prefixes = deps.map(&:to_formula).select(&:keg_only?).map(&:opt_prefix) + [HOMEBREW_PREFIX]
+    inreplace "Jamtop" do |s|
+      # These two inreplaces make sure all Homebrew libraries can be found by the Jamfile
+      s.gsub! %r{(\|\| \[ GLOB )/usr/local(/include\$\(subd\) :[^\]]*\])}, prefixes.map { |p| "\\1#{p}\\2" }.join(" ")
+      s.gsub! %r{(\|\| \[ GLOB )/usr/local(/lib : lib\$\(lcase\)\.a \])}, prefixes.map { |p| "\\1#{p}\\2" }.join(" ")
+      if OS.linux?
+        # These two inreplaces make sure the X11 headers can be found on Linux.
+        s.gsub! "/usr/X11R6/include", HOMEBREW_PREFIX/"include"
+        s.gsub! "/usr/X11R6/lib", HOMEBREW_PREFIX/"lib"
+      elsif (sdk = MacOS.sdk_path_if_needed)
+        # These two inreplaces make sure system zlib can be found on macOS.
+        s.gsub! "GLOB /usr/include$(subd) ", "GLOB #{sdk}/usr/include$(subd) "
+        s.gsub! "GLOB /usr/lib : lib$(lcase).so ", "GLOB #{sdk}/usr/lib : lib$(lcase).tbd "
+      end
     end
+
+    ENV["NUMBER_OF_PROCESSORS"] = ENV.make_jobs.to_s
```

Comment:
May end up adding something like following (or specific for `jpeg-turbo`) to avoid accidentally picking up `jpeg` if installed:
```suggestion
    ENV["NUMBER_OF_PROCESSORS"] = ENV.make_jobs.to_s
    ENV["LINKFLAGS"] = prefixes.map { |p| "-L#{p}/lib" }.join(" ")
```

When testing locally on Linux, `ld` complained about picking up `jpeg` library.

Looks like Jam file ends up adding `-ljpeg ...` flags before shim scripts add `-L#{Formula[...].opt_lib} ...` flags, which may be tripping up library detection on GCC-5.